aggregator_api/src/lib.rs

```diff
@@ -113,13 +115,23 @@ async fn auth_check(conn: &mut Conn, State(cfg): State<Arc<Config>>) -> impl Han
         constant_time::verify_slices_are_equal(bearer_token.as_ref(), key.as_ref()).is_ok()
     }) {
         // Authorization succeeds.
+        conn.set_state(cfg);
         return None;
     }
 
     // Authorization fails.
     Some((Status::Unauthorized, Halt))
 }
 
+async fn get_config(_: &mut Conn, State(config): State<Arc<Config>>) -> Json<AggregatorApiConfig> {
+    Json(AggregatorApiConfig {
+        dap_url: config.public_dap_url.clone(),
+        role: AggregatorRole::Either,
+        vdafs: vec![1, 2, 3],
```

Comment:
There's `VdafInstance` in `core/src/task.rs` for this, unfortunately you'd have to construct the enum instances to get its discriminant (likewise for `taskprov::VdafType`)

If that's too ugly, I'd still prefer we define a model (similar to AggregatorRole) to assign these numbers some meaning.

Reply:
Addressed in 05026e5e02500027dfc843bc0710789b4c8d595a